machine-runner/src/runner/runner.ts

```diff
@@ -932,6 +972,11 @@ export interface StateOpaque<
    * }
    */
   cast(): State<StateName, Payload, Commands>
+
+  /**
+   * Return true when the commands of its state counterpart is available; otherwise return false.
+   */
+  commandAvailable(): boolean
```

Comment:
I’d rename this to `commandsAvailable` to match the description

Reply:
renamed